### PR TITLE
Add token details

### PR DESCRIPTION
**Pull request by netanelC**
This PR does:
1. makes the names shorter
2. adds the jwt token parser
3. syncs with raster the helm

### CODE REVIEW

File: config/nginx_root.conf [hunk 1]
```diff
@@ -14,9 +15,6 @@ http {
     include       /etc/nginx/mime.types;
     default_type  application/octet-stream;
 
-    access_log /var/log/nginx/access.log;
```
Comment (RonitKissis) [inline config/nginx_root.conf:17]: why did you remove this? this is for the elastic logs

File: config/policy.rego [hunk 2]
```diff
@@ -9,36 +9,49 @@ default allow = false
 
 ### Resources Access ###
 user_has_resource_access[payload] {
-  lower(payload.d[_]) = lower(map_colonies.domain)
+	lower(payload.d[_]) = lower(map_colonies.domain)
```
Comment (RonitKissis) [inline config/policy.rego:12]: what is this tab for?

File: config/policy.rego [hunk 3]
```diff
@@ -9,36 +9,49 @@ default allow = false
...
 }
 ### Resources Access ###
 
 ### ORIGIN ###
 # Checks if origin is in allowed origin
 valid_origin[payload] {
-  payload.ao[_] = http_request.headers.origin
+	payload.ao[_] = http_request.headers.origin
```
Comment (RonitKissis) [inline config/policy.rego:19]: what is this tab for?

File: config/policy.rego [hunk 4]
```diff
@@ -9,36 +9,49 @@ default allow = false
...
 }
 
 # Checks if origin is allowed origin (if ao is not an arr)
 valid_origin[payload] {
-  payload.ao == http_request.headers.origin
+	payload.ao == http_request.headers.origin
```
Comment (RonitKissis) [inline config/policy.rego:24]: what is this tab for?

File: config/policy.rego [hunk 5]
```diff
@@ -9,36 +9,49 @@ default allow = false
...
 }
 
 # Checks if there is allowed origin
+valid_referrer[payload] {
```
Comment (RonitKissis) [inline config/policy.rego:28]: didnt we remove the origin becuase of issues we had in outnet?

File: config/policy.rego [hunk 6]
```diff
@@ -9,36 +9,49 @@ default allow = false
...
+	payload.ao[_] = http_request.headers.referrer
+}
+
+valid_referrer[payload] {
+	payload.ao == http_request.headers.referrer
+}
+
 valid_origin[payload] {
-  not payload.ao
+	not payload.ao
```
Comment (RonitKissis) [inline config/policy.rego:37]: what is this tab for?

**Review by RonitKissis — CHANGES_REQUESTED** [review]
did you deployed it in openshift and saw that everything is running correctly?

File: config/nginx_root.conf (hunk 1 above)
Comment (netanelC) [reply config/nginx_root.conf:17]: Didn't know, apologizing

File: config/policy.rego (hunk 2 above)
Comment (netanelC) [reply config/policy.rego:12]: I just synced from pycsw/helm/raster so we won't be behind. This doesn't mean a thing

File: config/policy.rego (hunk 3 above)
Comment (netanelC) [reply config/policy.rego:19]: I just synced from pycsw/helm/raster so we won't be behind. This doesn't mean a thing

File: config/policy.rego (hunk 4 above)
Comment (netanelC) [reply config/policy.rego:24]: I just synced from pycsw/helm/raster so we won't be behind. This doesn't mean a thing

File: config/policy.rego (hunk 6 above)
Comment (netanelC) [reply config/policy.rego:37]: I just synced from pycsw/helm/raster so we won't be behind. This doesn't mean a thing

File: config/policy.rego (hunk 5 above)
Comment (netanelC) [reply config/policy.rego]: This is something else that Raster added due to one client. Turned out they don't even use it... I removed it

**Review by RonitKissis — APPROVED** [review]

**Pull request merged**